src/Test/Perf/Runner/Program.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.IO;
+using Roslyn.Test.Performance.Utilities;
+using static Roslyn.Test.Performance.Utilities.TestUtilities;
+using static Roslyn.Test.Performance.Runner.Tools;
+using static Roslyn.Test.Performance.Runner.Benchview;
+using static Roslyn.Test.Performance.Runner.TraceBackup;
+using Roslyn.Test.Performance.Runner;
+
+namespace Runner
+{
+    class Program
```

Comment:
I went through all the files adding visibility modifers.  I also made all of our utilities classes `static class`es.